app/controllers/expenses_additional_sources_controller.rb

```diff
@@ -6,11 +6,7 @@ class ExpensesAdditionalSourcesController < StandardStepsController
   def step_params
     super.tap do |params_hash|
       params_hash.each do |key, val|
-        params_hash[key] = if val == "true"
-                             true
-                           else
-                             false
-                           end
+        params_hash[key] = val == "true"
```

Comment:
I'm not sure I follow this...

Reply:
this happened via rubocop's autocorrect. here it is in action:

```
params_hash = { test: "true" }
[10] pry(main)> params_hash.each do |key, val|
[10] pry(main)*   params_hash[key] = val == "true"
[10] pry(main)* end
=> {:test=>true}
[11] pry(main)> params_hash = { test: "blah" }
=> {:test=>"blah"}
[12] pry(main)> params_hash.each do |key, val|
[12] pry(main)*   params_hash[key] = val == "true"
[12] pry(main)* end
=> {:test=>false}
```

Reply:
agree that it's more confusing than the former!

Reply:
basically, it's evaluating `val == "true"` and setting the param key to whatever the result of that evaluation is (which is true or false)